recipes/hippomocks/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+from conans import ConanFile, ConanFile, tools
+import os, glob
+
+class HippomocksConan(ConanFile):
+    name = 'hippomocks'
+    version = '10960d0' # This library is not versioned
```

Comment:
```suggestion
```
Version should be declared on recipe file

Please, read and follow: master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
thanks a lot, I missed this info